core/contract.go

```diff
@@ -0,0 +1,49 @@
+package core
+
+import (
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+// Class unambiguously defines a [Contract]'s semantics.
```

Comment:
What do the `[]` square brackets mean?

Reply:
When the documentation is rendered as HTML it will link to the `Contract` struct.